Src/CSharpier.Cli/IgnoreFile.cs

```diff
@@ -7,6 +7,7 @@ public class IgnoreFile
 {
     protected Ignore.Ignore Ignore { get; }
     protected string IgnoreBaseDirectoryPath { get; }
+    private static string[] alwaysIgnored = { "**/node_modules/*.cs" };
```

Comment:
@SubjectAlpha @shocklateboy92 I think the correct format is `**/node_modules/**/*.cs`

Using v0.20.0 I got this formatting warning
```
Warning ./node_modules/selenium-webdriver/lib/test/data/Page.aspx.cs - Was not formatted.
```